lib/monkeypatch.js

```diff
@@ -0,0 +1,8 @@
+if (process.env.RUNNING_IN_SPECTRON) {
```

Comment:
I'm not completely sure what this file does; could you maybs add a tiny comment for future reference? Thanks 🙏